### ISSUE TITLE
Requesting Yerushalmi Shekalim by Vilna daf is 1a-indexed, not 2a-index

### ISSUE THREAD

**Issue by ronshapiro** [Sefaria/Sefaria-Project#628]
https://www.sefaria.org/api/texts/Jerusalem_Talmud_Shekalim.1a returns the text that appears on Vilna 1a (you can even see the "2a" superscript at the top). Requesting 2a seems to really be 3a, etc. But on the website this is fine. Any idea what's going on here?

**Comment by JonMosenkis** [comment]
This is a particularly confusing situation, as there are two "Vilna" divisions of the Yerushalmi.

Generally, when referring to "Vilna", one means the Vilna printing of the Yerushalmi, which goes 1a-33a. This is different than what appears within the Vilna Bavli printing, which goes from 2a to 22b. The daf yomi calendars follow the printing that appears within the Bavli, not the Vilna Yerushalmi.

We refer to the division that appears within the Bavli as "Daf Yomi". This is to differentiate between a future division we will add that will be known as "Vilna".

Regardless, the best way to request any data from Jerusalem Talmud Shekalim is to make a request using the Vilna Chapter-Halakha division. A mapping for that division to the daily daf can be obtained from the "Daf Yomi" alt-struct. Asking for Jerusalem Talmud 1a (or any other daf) will be ambiguous, as plan on adding both daf divisions in the near future.

**Comment by JonMosenkis** [comment]
You are correct though that the api is returning the wrong address. I stand by my previous comment though - requesting Shekalim by daf will lead to an ambiguous result in the near future.

**Comment by ronshapiro** [comment]
Thanks for the explanation!!

בתאריך יום ה׳, 11 במרץ 2021, 17:00, מאת JonMosenkis ‏<
***@***.***>:
